ShiftLogger.Bata13/STUDY.ASP.ShiftLoggerTryThree/Data/DataContext.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.EntityFrameworkCore;
+using STUDY.ASP.ShiftLoggerTryThree.Models;
+
+namespace STUDY.ASP.ShiftLoggerTryThree.Data
+{
+    public class DataContext : DbContext
+    {
+        public DataContext(DbContextOptions<DataContext> options) : base(options)
+        {
+
+        }
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            base.OnConfiguring(optionsBuilder);
+            optionsBuilder.UseSqlServer("Data Source=(LocalDB)\\LocalDBDemo;Database=shiftloggerdb;Trusted_Connection=true;TrustServerCertificate=true;");
```

Comment:
Just a little advice for reviewers comfort: Keep things like ConnectionString in appsetting.json or app.config files 😁 Those are first files reviewer opens to customize things for running project purpose